example/viam_example_app/lib/login_screen.dart

```diff
@@ -36,8 +36,8 @@ class _LoginScreenState extends State<LoginScreen> {
   /// This method will get called when the widget initializes its state.
   /// It exists outside the overridden [initState] function since it's async.
   Future<void> _initState() async {
-    // Load the `.env` file
-    await dotenv.load();
+    // Load the `.env` file TODO replace `filename` with your .env filename
+    await dotenv.load(fileName: 'YOUR_ENV_FILE.env');
```

Comment:
I think requiring a name for the file is a bit more complicated than just using `.env`. Especially because then they also have to update the `pubspec.yaml` to include their `.env` file (instead of using the default)

Reply:
hmmm i think i am doing this differently than you intended then. I didn't realize you could leave it as the default. You're totally right.